### ISSUE TITLE
Allow multiple custom on-the-fly inline validators per validation object

### ISSUE THREAD

**Issue by alexdiliberto** [anandts88/ember-validator#13]
The following restricts each validation object to allowing a maximum of one `custom` property. We should expand this restriction to allow the ability to add multiple custom on-the-fly validators per validation object.

https://github.com/anandts88/ember-validator/blob/f8e806bb7a836a856cfecd505a0ad1fef886e45d/addon/mixin.js#L412

**Comment by anandts88** [comment]
@alexdiliberto Custom validation is performed with the help of inlineValidator function which takes callback as an argument, we need to write our own logic inside this callback and return error message in case of failure scenario. Inside this callback we can write multiple custom logics and return error message for each logic

**Comment by alexdiliberto** [comment]
@anandts88 Yes I'm doing this as a solution but I think it would be a nice feature to have the ability to add multiple logical instances of custom validations simply as an enhancement:

``` js
let validationObj = {
  password: {
    required: { message: "Please enter password." },
    customPasswordReq1: inlineValidator(function(model, property) {
      return 'Custom Validation #1';
    }),
    customPasswordReq2: inlineValidator(function(model, property) {
      return 'Custom Validation #2';
    })
  },
};
```

This would allow the user to add multiple custom validations that would "just work" if both validation scenarios failed instead of having to manually have cases in a callback to handle multiple error scenarios. Also I think it composes better and is easier to read instead of having to look at conditionals within a callback. Even just a naming convention starts-with RegEx would work like `/^custom/` then if you say `customFoo` it know it's a custom validation.
